co2eq/purchase/index.js

```diff
@@ -154,7 +154,38 @@ function extractEur({ costAmount, costCurrency }) {
   return costAmount && costCurrency ? convertTo2011Euro(costAmount, costCurrency) : null;
 }
 
-function conversionCPI(eurAmount, referenceYear, countryCodeISO2, datetime) {
+export function getClosestYear(data, year) {
+  return Object.keys(data).reduce((prev, curr) =>
+    Math.abs(curr - year) <= Math.abs(prev - year) ? curr : prev
```

Comment:
Wondering why we need the `Math.abs`?

Reply:
Dunno, math? 😅 
Otherwise it would return a negative value (`2010-2016 = -6`), but we just want the difference (`Math.abs(2010-2016) = 6`).


Without `Math.abs`:

![image](https://user-images.githubusercontent.com/3296643/104321017-5aa4ef00-54e3-11eb-9b94-60fcdf2ce9c6.png)

With `Math.abs`:

![image](https://user-images.githubusercontent.com/3296643/104321032-61336680-54e3-11eb-887e-90f9be31738e.png)

Reply:
Right! Math...